code/modules/reagents/reagent_dispenser.dm

```diff
@@ -89,6 +91,20 @@
 		message_admins("[key_name_admin(Proj.firer)] triggered a fueltank explosion.")
 		log_game("[key_name(Proj.firer)] triggered a fueltank explosion.")
 		boom()
+		
+/obj/structure/reagent_dispensers/fueltank/attackby(obj/item/weapon/W, mob/user, params)
+	if(istype(W,/obj/item/device/assembly_holder))
+		if((W.name == "remote signaling device-igniter assembly") || (W.name == "igniter-remote signaling device assembly"))
+			if(!user.drop_item())
+				return
+			W.loc = src
+			user << "You place the igniter assembly inside the fuel tank, it's primed to blow on your signal!"
+		else
+			user << "The assembly won't fit into the tank! Only a signaller or igniter would be able to fit inside."
+	if(istype(W,/obj/item/device/assembly/igniter)
+		user << "You decide against tossing the igniter into the fuel tank, why bother if you have no way to signal it?"

```

Comment:
I'm going to do something about it, but there's no easy solutions.

I let you use a prox sensor, then you set it up and realize that since the beginning of time prox sensors won't get set off when they're inside something else (I think this is still true). For now I'm just going to remove the name check, but roughly half of igniter assemblies will not work as you might intend them.